app/src/main/java/org/cru/godtools/ui/profile/MyProfileActivity.kt

```diff
@@ -0,0 +1,66 @@
+package org.cru.godtools.ui.profile
+
+import android.app.Activity
+import android.content.Intent
+import android.os.Bundle
+import androidx.fragment.app.FragmentActivity
+import androidx.viewpager2.adapter.FragmentStateAdapter
+import com.google.android.material.tabs.TabLayoutMediator
+import me.thekey.android.TheKey
+import org.cru.godtools.R
+import org.cru.godtools.activity.BasePlatformActivity
+import org.cru.godtools.analytics.model.AnalyticsScreenEvent
+import org.cru.godtools.analytics.model.AnalyticsScreenEvent.Companion.SCREEN_GLOBAL_DASHBOARD
+import org.cru.godtools.base.ui.activity.BaseActivity
+import org.cru.godtools.databinding.ActivityMyProfileBinding
+
+fun Activity.startMyProfileActivity() {
+    Intent(this, MyProfileActivity::class.java)
+        .putExtras(BaseActivity.buildExtras(this))
+        .also { startActivity(it) }
+}
+
+class MyProfileActivity : BasePlatformActivity() {
+
+    private var binding: ActivityMyProfileBinding? = null
```

Comment:
in an Activity the view is only created once (unless you trigger `setContentView` other times during an Activity's lifespan).

because of that you don't need to make the binding nullable in an activity unless you have use cases that won't create the binding.